restructure pricing page (#5193)
getsentry/sentry-docs: src/docs/product/accounts/pricing.mdx

- **Shared:** Your on-demand budget is shared among errors, transactions, and attachments.

- **Per-category:** Separate on-demand budgets for errors, transactions, and attachments. Any overages in one category will not consume the budget of another category.

![On-Demand Budgets settings in Subscriptions](on-demand-budgets.png)

If you raise your on-demand budget mid-month, your organization will start accepting additional data as soon as your new budget is applied. If you lower your on-demand budget mid-month to below the current month's usage, your on-demand budget will be reduced to match your usage. On your next bill date, your on-demand budget will be lowered to the requested amount.

After a new budget is in effect, all additional data will be rejected and no additional on-demand volume will be added. At the end of the billing month, you will be charged for any on-demand volume consumed. We guarantee new budgets will be applied and effective within 24 hours. However, in most cases the new budget will be applied within minutes.

## Pricing: How it Works

Our pricing model is similar to the way that you purchase data for a cell phone:

> - We bill you for [prepaid volume](#prepaid-reserved-capacity) monthly or annually

Author: ```suggestion
- We bill you for [prepaid volume](#prepaid-reserved-capacity) monthly or annually.
```

> - You can also set up an [on-demand budget](#on-demand-capacity), for possible overages, which is priced differently.

Author: ```suggestion
- You can also set up an [on-demand budget](#on-demand-capacity) for possible overages, which is priced differently.
```

|---|---|---|
| >25 - 100 (GB) | $0.24 | $0.312 |
| >100 - 200 (GB) | $0.24 | $0.312 |
| >200 - 10k (GB) | $0.23 | $0.299 |

<!-- prettier-ignore-end -->

You can buy prepaid volume in increments as shown in the table below:

<!-- prettier-ignore-start -->

<!-- Don't remove spaces from first column header or number ranges in column will break onto two lines making them hard to read -->

| Errors Volume&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; | Purchase Increment | Transaction Volume | Purchase Increment | Attachments Volume | Purchase Increment |
|---|---|---|---|---|---|
| >50k-100k | 50k | >100k-250k | 150k | >25 - 100 (GB) | 25 (GB) |
| >100k-500k | 100k | >250k-500k | 250k | >100 - 10k (GB) | 100 (GB) |
| >500k-2M | 500k | >500k-5M | 500k |  |  |
| >2M-20M | 1M | >5M-10M | 1M |  |  |
| >20M-50M | 2M | >10M-20M | 2M |  |  |

<!-- prettier-ignore-end -->

On-demand purchasing isn't restricted by any increments since you set a dollar amount for your budget, rather than a certain amount of data. So if you want to purchase an amount of prepaid volume data that falls between increments, you should set that up as on-demand budget.

> The slider on the [pricing](https://sentry.io/pricing/) page allows you to choose the prepaid volume you'd like for each type of event (errors and transactions) or attachments storage to see what you'd pay.

Author: ```suggestion
The slider on the [pricing](https://sentry.io/pricing/) page allows you to choose the prepaid volume you'd like for each type of event (errors, transactions) or attachments storage to see what you'd pay.
```

---
title: Pricing & Billing
sidebar_order: 45
redirect_from:
  - /pricing/
  - /learn/pricing/
  - /product/pricing/
description: "Learn about pricing variables, managing volume, and different Sentry plans."
---

This document clarifies concepts around pricing and billing. For the breakdown of plans and prices, go to our [pricing page](https://sentry.io/pricing/).

## Volume {#capacity}

> Volume is the total number of [events](/product/sentry-basics/key-terms/) and attachments or the total amount of [data](/product/sentry-basics/key-terms/) sent by and billed to your organization. You can purchase volume in two ways: [prepaid (or reserved)](#prepaid-reserved-capacity) and [on-demand](#on-demand-capacity).

Author: ```suggestion
Volume is the total number of [events](/product/sentry-basics/key-terms/) and attachments — or the total amount of [data](/product/sentry-basics/key-terms/) — sent by and billed to your organization. You can purchase volume in two ways: [prepaid (or reserved)](#prepaid-reserved-capacity) and [on-demand](#on-demand-capacity).
```

### Prepaid (Reserved) Volume {#prepaid-reserved-capacity}

Reserved volume (or quota) allows you to prepay for data at significant discounts. Your prepaid, reserved volume is always applied before on-demand volume. Unused reserved volume expires at the end of each billing month.

Every Sentry account includes a varying amount of reserved volume. For example, our Team plan includes 50k errors, 100k transactions, and one GB of attachments per month:

![Reserved Volume settings in Subscriptions](reserved-volume.png)

To see prepaid pricing per volume tier, check out our [pricing tables](#pricing-tables).

### On-Demand Volume {#on-demand-capacity}

> On-demand volume is billed at the end of each billing cycle. You are charged per processed event, rounded up to the nearest cent. You have the ability to set on-demand caps (or quotas) so you can stay on budget. Your on-demand volume is determined by your on-demand budget, which can be configured on either a shared or per-category (errors, transactions, attachments) basis. Learn more in [On-Demand Budgets](#on-demand-cap).

Author: Nope, it's a good edit!

> On-demand volume is billed at the end of each billing month. You are charged per processed event, rounded up to the nearest cent. You have the ability to set on-demand budgets (or quotas) so you can stay on budget. Your on-demand volume is determined by your on-demand budget, which can be configured on either a shared or per-category basis. Learn more in [On-Demand Budgets](#on-demand-cap).

Author: ```suggestion
On-demand volume is billed at the end of each billing month. You are charged per processed event or attachment, rounded up to the nearest cent. You have the ability to set on-demand budgets (or quotas) so you can stay on budget. Your on-demand volume is determined by your on-demand budget, which can be configured on either a shared or per-category basis. Learn more in [On-Demand Budgets](#on-demand-cap).
```

> To see on-demand pricing per volume tier, check out our [pricing table](#pricing-tables).

Author: ```suggestion
To see on-demand pricing per volume tier, check out our [pricing tables](#pricing-tables).
```

> To see on-demand pricing per volume tier, check out our [pricing table](#is-there-detailed-information-for-each-plan).

Author: ```suggestion
To see on-demand pricing per volume tier, check out our [pricing table](#pricing-tables).
```

#### On-Demand Budgets {#on-demand-cap}

You can set a maximum monthly on-demand invoice amount by setting an on-demand budget. Once you reach the maximum budget volume, you will still be able to access prior event data. However, any additional events you send will be rejected.

If you raise your on-demand budget mid-month, your organization will start accepting additional events as soon as your new budget is applied. We guarantee new budgets will be applied and effective within 24 hours. However, in most cases your organization will start accepting events within minutes.

> If you lower your on-demand budget mid-month below the month’s existing bill, we guarantee your new, lowered on-demand budget will be applied within 24 hours. In the meantime, the old on-demand budget will remain in effect. However, in most cases, the new budget will be applied within minutes.

Author: This is definitely better. Priyanka mentioned this to me and I forgot about it.

---
title: Pricing & Billing

> sidebar_order: 60

Author: ```suggestion
sidebar_order: 45
```
```suggestion
sidebar_order: 60
```

description: "Learn about pricing variables, managing event volume, and different Sentry plans."
---

This document clarifies concepts around pricing and billing. For the breakdown of plans and prices, go to our [pricing page](https://sentry.io/pricing/).

## Volume {#capacity}

Volume is the total number of [events](/product/sentry-basics/key-terms/) and attachments or the total amount of [data](/product/sentry-basics/key-terms/) sent by and billed to your organization. You can purchase volume in two ways: [prepaid (or reserved)](#prepaid-reserved-capacity) and [on-demand](#on-demand-capacity).

### Prepaid (Reserved) Volume {#prepaid-reserved-capacity}

Reserved volume (or quota) allows you to prepay for data at significant discounts. Your prepaid, reserved volume is always applied before on-demand volume. Unused reserved volume expires at the end of each billing month.

Every Sentry account includes a varying amount of reserved volume. For example, our Team plan includes 50k errors, 100k transactions, and one GB of attachments per month:

![Reserved Volume settings in Subscriptions](reserved-volume.png)

> To see prepaid pricing per volume tier, check out our [pricing table](#pricing-table).

Author: ```suggestion
To see prepaid pricing per volume tier, check out our [pricing table](#pricing-tables).
```
```suggestion
To see prepaid pricing per volume tier, check out our [pricing table](#pricing-table).
```

### On-Demand Volume {#on-demand-capacity}

On-demand volume is billed at the end of each billing cycle. You are charged per processed event, rounded up to the nearest cent. You have the ability to set on-demand caps (or quotas) so you can stay on budget. Your on-demand volume is determined by your on-demand budget, which can be configured on either a shared or per-category (errors, transactions, attachments) basis. Learn more in [On-Demand Budgets](#on-demand-cap).

> To see on-demand pricing per volume tier, check out our [pricing table](#pricing-table).

Author: ```suggestion
To see on-demand pricing per volume tier, check out our [pricing table](#is-there-detailed-information-for-each-plan).
```
```suggestion
To see on-demand pricing per volume tier, check out our [pricing table](#pricing-table).
```

## Pricing: How it Works {#faq}

Our pricing model is similar to the way that you purchase data for a cell phone:

- We bill you for [prepaid volume](#prepaid-reserved-capacity) every month
- You can also set up an [on-demand budget](#on-demand-capacity), for possible overages, which is priced differently.

You're only billed for on-demand volume if you use it. If you run out of prepaid data and on-demand budget, then we drop events and attachments and you're not billed for that data.

To determine your on-demand pricing, start at the tier where your prepaid purchasing ends. For example, you've purchased 500k of reserved errors with an on-demand budget of $25 on a Team plan. If you use 600k errors, you would pay for 500k prepaid errors at $0.000175/each and 100k on-demand errors at $0.000195/each, using $19.50 of your on-demand budget.

In another scenario, you've purchased 400k reserved errors with an on-demand budget of $25 on a Team plan. If you use 500k errors in total, you you would pay for 400k prepaid errors at $0.000175/each and 100k on-demand errors at $0.000228/each, using $22.80 of your on-demand budget.

> ### Pricing Tables {#is-there-detailed-information-for-each-plan}

Author: ```suggestion
### Pricing Tables 
```

| >500k-2M | $0.000150 | $0.000195 | $0.000300 | $0.000390 |
| >2M-10M | $0.000150 | $0.000195 | $0.000300 | $0.000390 |
| >10M-20M | $0.000130 | $0.000169 | $0.000251 | $0.000326 |
| >20M-50M | $0.000110 | $0.000143 | $0.000132 | $0.000172 |

#### Transactions Pricing

| Total Transaction Volume | Team Prepaid | Team On-Demand | Business Prepaid | Business On-Demand |
|---|---|---|---|---|
| >100k-250k | $0.000100 | $0.000130 | $0.000300 | $0.000390 |
| >250k-500k | $0.000058 | $0.000075 | $0.000178 | $0.000231 |
| >500k-4.5M | $0.000052 | $0.000068 | $0.000130  | $0.000169 |
| >4.5M-5M | $0.000046 | $0.000060 | $0.000092 | $0.000120 |
| >5M-10M | $0.000046 | $0.000060 | $0.000092 | $0.000120 |
| >10M-20M | $0.000044 | $0.000057 | $0.000077 | $0.000100 |
| >20M-50M | $0.000042 | $0.000055 | $0.000074 | $0.000096 |

#### Attachments Pricing

| Total Attachments Volume | Team/Business Prepaid | Team/Business On-Demand |
|---|---|---|
| >25 - 100 (GB) | $0.24 | $0.312 |
| >100 - 200 (GB) | $0.24 | $0.312 |
| >200 - 10k (GB) | $0.23 | $0.299 |

<!-- prettier-ignore-end -->

You can buy prepaid volume in increments as shown in the table below:

<!-- prettier-ignore-start -->

> | Errors Volume | Purchase Increment | Transaction Volume | Purchase Increment | Attachments Volume | Purchase Increment |

Author: ```suggestion
<!-- Don't remove spaces from first column header or number ranges in column will break onto two lines making them hard to read -->

| Errors Volume&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; | Purchase Increment | Transaction Volume | Purchase Increment | Attachments Volume | Purchase Increment |
```

> | Errors Volume&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; | Purchase Increment | Transaction Volume | Purchase Increment | Attachments Volume | Purchase Increment |

Author: When I was testing this locally, without them, the number ranges in that column broke onto two lines, which made them hard to read. But I'll test it in the preview without them, just to be sure.

> | Errors Volume&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; | Purchase Increment | Transaction Volume | Purchase Increment | Attachments Volume | Purchase Increment |

Author: ```suggestion
| Errors Volume | Purchase Increment | Transaction Volume | Purchase Increment | Attachments Volume | Purchase Increment |
```

> | Errors Volume&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; | Purchase Increment | Transaction Volume | Purchase Increment | Attachments Volume | Purchase Increment |

Author: Turns out they're load bearing!

|---|---|---|---|---|---|
| >50k-100k | 50k | >100k-250k | 150k | >25 - 100 (GB) | 25 (GB) |
| >100k-500k | 100k | >250k-500k | 250k | >100 - 10k (GB) | 100 (GB) |
| >500k-2M | 500k | >500k-5M | 500k |  |  |
| >2M-20M | 1M | >5M-10M | 1M |  |  |
| >20M-50M | 2M | >10M-20M | 2M |  |  |

<!-- prettier-ignore-end -->

By contrast, on-demand purchasing isn't restricted by any increments since you set a dollar amount for your budget, rather than a certain amount of data. So if you want to purchase an amount of prepaid volume data that falls between increments, you should set that up as on-demand budget.

The slider on the [pricing](https://sentry.io/pricing/) page allows you to choose the prepaid volume you'd like for each type of event - errors, transactions, or attachments storage to see what you'd pay.

If your projects have a high volume of events, you can control how many errors Sentry receives. For more information on how to manage your monthly spending, check out our [Quota Management documentation](/product/accounts/quotas/).

> ## Billing Cycles

Author: ```suggestion
## Billing Periods {#billing-cycles}
```

The second is the _On-Demand Period_. The on-demand period reflects a one month window in which we calculate on-demand charges.

Users with the role of [Billing or Owner](/product/accounts/membership/) can find all previous invoices by going to **Organization > Usage & Billing > Usage & Payments > Receipts**.

<Note>

You must have either `Billing` or `Owner` permissions to access or make changes to information on this page. If you don't have those permissions, or you are not a member of the organization in Sentry, please reach out to a Billing member or an Owner to obtain invoices.

</Note>

## Plans & Free Trials

Sentry has one free Developer plan and three paid plans: Team, Business, and Enterprise. You can find more information about each of these plans and what they include on our [pricing page](https://sentry.io/pricing/).

Plan upgrades take effect immediately. Plan downgrades and cancellations are processed as the end of the current contract cycle and cannot be refunded. If you no longer want to be on a paid plan, you'll revert to the base Developer plan at the end of your current billing cycle.

> If you're currently on a paid plan, you can sign up for a 30-day trial; if you're on a free plan, you can run a trial for 14 days. Before the trial ends, Sentry will provide you options to upgrade and continue to use our latest features like Performance Monitoring. If you choose to opt in to a new plan at the end of your trial, the new plan takes effect immediately. If you don't sign up for plan or upgrade by the end of a trial, you will be downgraded to the same plan you were on prior to starting the trial.

Author: Makes sense to me!

- **Per-category:** Separate on-demand budgets for errors, transactions, and attachments. Any overages in one category will not consume the budget of another category.

![On-Demand Budgets settings in Subscriptions](on-demand-budgets.png)

## Pricing: How it Works

Our pricing model is similar to the way that you purchase data for a cell phone:

> - We bill you for [prepaid volume](#prepaid-reserved-capacity) every month

Author: ```suggestion
- We bill you for [prepaid volume](#prepaid-reserved-capacity) monthly or annually
```

> - We bill you for [prepaid volume](#prepaid-reserved-capacity) every month

Author: I've changed it to monthly or annually, but I'm gonna leave out the 10% part since that's a bit more nitty gritty than I'm looking to go on this

|---|---|---|
| >25 - 100 (GB) | $0.24 | $0.312 |
| >100 - 200 (GB) | $0.24 | $0.312 |
| >200 - 10k (GB) | $0.23 | $0.299 |

<!-- prettier-ignore-end -->

You can buy prepaid volume in increments as shown in the table below:

<!-- prettier-ignore-start -->

<!-- Don't remove spaces from first column header or number ranges in column will break onto two lines making them hard to read -->

| Errors Volume&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; | Purchase Increment | Transaction Volume | Purchase Increment | Attachments Volume | Purchase Increment |
|---|---|---|---|---|---|
| >50k-100k | 50k | >100k-250k | 150k | >25 - 100 (GB) | 25 (GB) |
| >100k-500k | 100k | >250k-500k | 250k | >100 - 10k (GB) | 100 (GB) |
| >500k-2M | 500k | >500k-5M | 500k |  |  |
| >2M-20M | 1M | >5M-10M | 1M |  |  |
| >20M-50M | 2M | >10M-20M | 2M |  |  |

<!-- prettier-ignore-end -->

By contrast, on-demand purchasing isn't restricted by any increments since you set a dollar amount for your budget, rather than a certain amount of data. So if you want to purchase an amount of prepaid volume data that falls between increments, you should set that up as on-demand budget.

> The slider on the [pricing](https://sentry.io/pricing/) page allows you to choose the prepaid volume you'd like for each type of event - errors, transactions, or attachments storage to see what you'd pay.

Author: ```suggestion
The slider on the [pricing](https://sentry.io/pricing/) page allows you to choose the prepaid volume you'd like for each type of event — errors and transactions — or attachments storage to see what you'd pay.
```

> If your projects have a high volume of events, you can control how many errors Sentry receives. For more information on how to manage your monthly spending, check out our [Quota Management documentation](/product/accounts/quotas/).

Reviewer: Nit: I'm a little confused whether "events" refers to attachments or if the term "data" should be used instead. I'm seeing many examples of "events" referring to attachments in this doc, but that's not consistent with the definition in `key-terms`.

Author: Events is meant to = errors and transactions, while data encompasses all the possible categories. I'll do a sweep of this doc again to make sure I've caught all the mentions.

Author: I've gone through and changed any instances of the word "events" to either "events and attachments" or "data" depending on which made more sense.

Users with the role of [Billing or Owner](/product/accounts/membership/) can find all previous invoices by going to **Organization > Usage & Billing > Usage & Payments > Receipts**.

<Note>

You must have either `Billing` or `Owner` permissions to access or make changes to information on this page. If you don't have those permissions, or you are not a member of the organization in Sentry, please reach out to a Billing member or an Owner to obtain invoices.

</Note>

## Plans & Free Trials

Sentry has one free Developer plan and three paid plans: Team, Business, and Enterprise. You can find more information about each of these plans and what they include on our [pricing page](https://sentry.io/pricing/).

Plan upgrades take effect immediately. Plan downgrades and cancellations are processed as the end of the current contract cycle and cannot be refunded. If you no longer want to be on a paid plan, you'll revert to the base Developer plan at the end of your current billing cycle.

> If you're currently on a paid plan, you can sign up for a 30-day trial; if you're on a free plan, you can run a trial for 14 days. Before the trial ends, Sentry will provide you options to upgrade and continue to use our latest features like Performance Monitoring. If you choose to opt in to a new plan during your trial, or at the end of your trial, the new plan takes effect immediately. If you don't sign up for plan or upgrade by the end of a trial, you will be downgraded to the same plan you were on prior to starting the trial.

Reviewer: ```suggestion
If you're currently on a paid plan, you can sign up for a 30-day trial; if you're on a free plan, you can run a trial for 14 days. Before the trial ends, Sentry will provide you options to upgrade and continue to use our latest features like Performance Monitoring. If you choose to opt in to a new plan during your trial, or at the end of your trial, the new plan takes effect immediately. If you don't upgrade by the end of trial, you will be downgraded to the same plan you were on prior to starting the trial.
```

- Utah
- Washington

Currently, prices displayed on our [pricing page](http://sentry.io/pricing) don't include sales tax since tax rates and applicability varies across country, state, and locality.

### Tax Exemptions

If your company or non-profit organization qualifies for a sales tax exemption, you can reach out to tax@sentry.io with the following information to have sales tax removed from any future invoices:

> - Organization Name

Author: ```suggestion
- Organization name
```

- Copy of a signed tax exempt certificate

If you've submitted an exemption form, and you're still being charged sales taxes, that's because either:

- The exemption certificate was still in review after the billing period was closed
- The exemption certificate covers a state that is different from your billing address

> Updating your address or submitting a tax exemption certificate, doesn't impact the taxes charged on older invoices. We do not adjust the billing or invoicing of a closed billing period.

Author: ```suggestion
Updating your address or submitting a tax exemption certificate doesn't impact the taxes charged on older invoices. We do not adjust the billing or invoicing of a closed billing period.
```
